src/Decomposer.php

```diff
@@ -133,6 +133,16 @@ public static function getComposerArray()
 
     public static function getPackagesAndDependencies($packagesArray)
     {
+        $packageLockDetails = [];
```

Comment:
The `$packageLockDetails` array is also initialized in the if loop below. We don't need to initialize it twice.